readme.md

```diff
@@ -332,6 +333,10 @@ _**Please note:** If you don't want the additional http request for the polyfill
 
 The Kirby chart components use Highcharts. Note that this is a licensed product.
 
+## Auto-merge
+
+Adding the `merge:ready` label to a pull-request will cause it to automatically be merged with `master` once it passes all checks (including approval of the pull-request). Once auto-merging begins the pull-request will automatically be labeled with the `automerge`-label.
```

Comment:
```suggestion
Adding the `merge:ready` label to a pull-request will cause it to be automatically labeled with the `automerge` label once it has a PR approval and is first in the queue of PRs with the `merge:ready` label ("first come, first served"). 
With the `automerge` label once the PR passes all checks (lint, test, build & PR approval) it will be automatically updated with latest master and finally merged back to master.
```